### PR TITLE
Adds MySQL Management example and refactors other DB samples

### PR DESCRIPTION
**Pull request by berndverst**

### CODE REVIEW

**Comment by berndverst** [comment]
cc @lmazuel @lisawong19

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/python-sdk-azure-authenticate.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-conceptual/python-sdk-azure-authenticate.md) | :white_check_mark:Succeeded | |
[docs-ref-services/mysql.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-services/mysql.md) | :white_check_mark:Succeeded | |
[docs-ref-services/postgresql.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-services/postgresql.md) | :white_check_mark:Succeeded | |
[docs-ref-services/sql.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-services/sql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C21%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707202356031240-78%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/mysql.md [hunk 1]
```diff
@@ -27,39 +27,87 @@ The recommended client library for accessing Azure Database for MySQL is the Mic
 Connect to a Azure Database for MySQL and select all records in the sales table. You can get the ODBC connection string for the database from the Azure Portal.
 
 ```python
-server = SERVER_NAME+'.mysql.database.azure.com'
-database = DATABASE_NAME
-username = USER_NAME
-password = PASSWORD
-driver = '{MySQL ODBC 5.3 UNICODE Driver}'
+SERVER = 'YOUR_SEVER_NAME.mysql.database.azure.com'
```
Comment (lmazuel) [inline docs-ref-services/mysql.md]: Do you mean to remove the concat here?

File: docs-ref-services/mysql.md [hunk 2]
```diff
@@ -27,39 +27,87 @@ The recommended client library for accessing Azure Database for MySQL is the Mic
...
 
 ## Management API
 
 Create and manage MySQL resources in your subscription with the management API.
 
+### Requirements
+You must install the MySQL management libraries for Python.
 ```bash
-pip install azure-mgmt-rdbms
+pip3 install azure-mgmt-rdbms
```
Comment (lmazuel) [inline docs-ref-services/mysql.md]: don't pip3, just pip

File: docs-ref-services/mysql.md [hunk 3]
```diff
@@ -27,39 +27,87 @@ The recommended client library for accessing Azure Database for MySQL is the Mic
...
+MYSQL_SERVER = "YOUR_DESIRED_MYSQL_SERVER_NAME"
+MYSQL_ADMIN_USER = "YOUR_MYSQL_ADMIN_USERNAME"
+MYSQL_ADMIN_PASSWORD = "YOUR_MYSQL_ADMIN_PASSOWRD"
+LOCATION = "westus"  # example Azure availability zone, should match resource group
+
+
+client = MySQLManagementClient(credentials=creds,
+    subscription_id=SUBSCRIPTION_ID)
+
+job_creation_poller = client.servers.create_or_update(
```
Comment (lmazuel) [inline docs-ref-services/mysql.md]: server_creation_poller

File: docs-ref-services/mysql.md [hunk 4]
```diff
@@ -27,39 +27,87 @@ The recommended client library for accessing Azure Database for MySQL is the Mic
...
+        ServerPropertiesForDefaultCreate(
+            administrator_login=MYSQL_ADMIN_USER,
+            administrator_login_password=MYSQL_ADMIN_PASSWORD,
+            version=ServerVersion.five_full_stop_seven
+        ),
+        location=LOCATION
+    )
+)
+
+server = job_creation_poller.result()
```
Comment (lmazuel) [inline docs-ref-services/mysql.md]: server_creation_poller

File: docs-ref-services/mysql.md [hunk 5]
```diff
@@ -27,39 +27,87 @@ The recommended client library for accessing Azure Database for MySQL is the Mic
...
+
+# Open access to this server for IPs
+job_creation_poller = client.firewall_rules.create_or_update(
```
Comment (lmazuel) [inline docs-ref-services/mysql.md]: rule_creation_poller

File: docs-ref-services/mysql.md [hunk 6]
```diff
@@ -27,39 +27,87 @@ The recommended client library for accessing Azure Database for MySQL is the Mic
...
+    RESOURCE_GROUP
+    MYSQL_SERVER,
+    "some_custom_ip_range_whitelist_rule_name",  # Custom firewall rule name
+    "123.123.123.123",  # Start ip range
+    "167.220.0.235"  # End ip range
+)
+
+firewall_rule = job_creation_poller.result()
```
Comment (lmazuel) [inline docs-ref-services/mysql.md]: rule_creation_poller

File: docs-ref-services/mysql.md [hunk 7]
```diff
@@ -27,39 +27,87 @@ The recommended client library for accessing Azure Database for MySQL is the Mic
...
+```
 
-### Samples
 
-* [Use Python to connect and query data][1]   
 
-[1]: https://docs.microsoft.com/azure/sql-database/sql-database-connect-query-python
+client = PostgreSQLManagementClient(credentials, SUBSCRIPTION_ID)
+
+job_creation_poller = client.databases.create_or_update(
```
Comment (lmazuel) [inline docs-ref-services/mysql.md]: db_creation_poller

File: docs-ref-services/mysql.md [hunk 8]
```diff
@@ -27,39 +27,87 @@ The recommended client library for accessing Azure Database for MySQL is the Mic
...
+    resource_group_name=RESOURCE_GROUP,
+    server_name=POSTGRES_SERVER, database_name=DB_NAME)
+job = job_creation_poller.result()
```
Comment (lmazuel) [inline docs-ref-services/mysql.md]: db = db_creation_poller.result()

File: docs-ref-services/mysql.md (hunk 3 above)
Comment (berndverst) [inline docs-ref-services/mysql.md]: Then why did you say job last time? :)

File: docs-ref-services/mysql.md (hunk 1 above)
Comment (berndverst) [inline docs-ref-services/mysql.md]: There was no concat before, but I'll add one.

File: docs-ref-services/mysql.md (hunk 2 above)
Comment (berndverst) [inline docs-ref-services/mysql.md]: Why not make Python3 the default?

**Comment by berndverst** [comment]
@lmazuel I addressed your comments

**Comment by sankethka** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/python-sdk-azure-authenticate.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-conceptual/python-sdk-azure-authenticate.md) | :white_check_mark:Succeeded | |
[docs-ref-services/mysql.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-services/mysql.md) | :white_check_mark:Succeeded | |
[docs-ref-services/postgresql.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-services/postgresql.md) | :white_check_mark:Succeeded | |
[docs-ref-services/sql.md](https://github.com/berndverst/azure-docs-sdk-python/blob/aug-1-stage/docs-ref-services/sql.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C21%5Ca92b76e3-3152-c0a9-537b-4d37096a01a1%5CPullRequest%5C201707210040567701-78%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**